lib/passenger.rb

```diff
@@ -17,8 +17,45 @@ def get_drivers
       @trips.map{ |t| t.driver }
     end
 
+    def get_total_money
+     total = 0
+     @trips.each do |trip|
+       total += trip.cost
+     end
+     return total
+    end
+
+    def total_time
+      total = 0
+      @trips.each do |trip|
+        total = trip.end_time - trip.start_time
+      end
+      return total
+    end
+
+
     def add_trip(trip)
       @trips << trip
     end
   end
 end
+
+
+ #pseudocode - describe "get total_rev do
```

Comment:
It's good that you're putting pseudocode here to describe what you're trying to do.